### DISCUSSION TITLE
Dynamic Driver Feed Window for Specific Race Positions

### DISCUSSION THREAD

**Discussion by justwillgreene** [f1multiviewer/issue-tracker#385] [Feature requests]
### Description:
Currently, the project allows users to open individual windows for specific driver feeds, which is valuable for monitoring a particular driver throughout the race. However, I would like to request the option to have a window dedicated to a specific race position, automatically updating to display the feed of the driver occupying that position at any given time.

### Use Cases:
By introducing dynamic driver feed windows tied to specific race positions, users can gain a broader perspective on the race. This feature would be particularly useful for tracking the performance of drivers in key positions, such as those battling for podium finishes or those leading the race. It would provide a more comprehensive viewing experience, allowing users to easily monitor the drivers in important positions without manually switching between individual driver feed windows.

### Implementation Ideas:
To implement this feature, I suggest adding a new window option within the program. Users can create a window and associate it with a specific race position, such as position 3. The program would then automatically update the window to display the feed of the driver currently occupying that position in real-time. This dynamic updating would enable users to stay informed about the driver in a specific race position without any manual intervention.

### Relevance to the Project:
Introducing dynamic driver feed windows tied to specific race positions aligns with the goal of providing a comprehensive and user-friendly viewing experience in the "MultiViewer for F1" project. By allowing users to effortlessly track drivers in specific positions, this enhancement would enhance the project's usability and enrich the analysis and enjoyment of Formula 1 races.

### Conclusion:
Thank you for considering my feature request. I believe that the addition of dynamic driver feed windows tied to specific race positions would greatly enhance the user experience and provide a valuable perspective on the race for "MultiViewer for F1" users. I appreciate your ongoing efforts and look forward to the future development of this fantastic project.

**Comment by jeppeolesen** [comment]
Marking this as a duplicate of #63
